src/features/plans/panels/TriggerPanel.tsx

```
@@ -251,41 +250,42 @@ export default function TriggerPanel({
+                        }}
+                        className="flex w-full items-center gap-3 rounded-lg border border-border/50 p-3 hover:bg-secondary/50 disabled:opacity-60"
+                      >
+                        <img
+                          src={place.thumbnailUrl ?? '/seoul_forest.jpg'}
+                          className="h-12 w-16 rounded-md object-cover"
+                        />
+                        <div>
+                          <p className="text-sm font-medium">{place.name}</p>
+                          <p className="text-xs text-muted-foreground">{place.location}</p>
+                        </div>
+                      </button>
```

Biome (2.3.14)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)